src/Bonsai.Lsl/Bonsai.Lsl.csproj

```diff
@@ -20,7 +20,7 @@
     <TargetFramework>net472</TargetFramework>
     <LangVersion>8.0</LangVersion>
     <Features>strict</Features>
-    <Version>0.2.0</Version>
+    <Version>0.2.1-rc1</Version>
```

Comment:
Since we are adding new features, I would bump the minor version to `0.3.0`.

Reply:
Changed in b7a21e20b9cc13eb9ade43d3157271b65ac68caa